cluster/mesos/README.md

```diff
@@ -8,33 +11,48 @@ containers on a Mesos cluster.
 
 ## Prerequisites
 
-Each node in your swarm must run a Mesos slave. The slave must be capable of starting tasks in a Docker Container using the `--containerizer=docker` option.
+Each node in your swarm must run a Mesos slave. The slave must be capable of
+starting tasks in a Docker Container using the `--containerizer=docker` option.
 
-You need to configure two TCP ports on the slave. One port to listen for the swarm manager, for example 2375. And a second TCP port to listen for the Mesos master, for example 3375.
+You need to configure two TCP ports on the slave. One port to listen for the
+swarm manager, for example 2375. And a second TCP port to listen for the Mesos
+master, for example 3375.
 
 ## Start the Docker Swarm manager
 
 If you use a single Mesos master:
 
 ```     
-docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 <mesos_master_ip>:<mesos_master:port>
+docker run -d -p <swarm_port>:2375 -p 3375:3375 \
+    swarm manage \
+        -c mesos-experimental \
+        --cluster-opt mesos.address=<public_machine_ip> \
+        --cluster-opt mesos.port=3375 \
+        <mesos_master_ip>:<mesos_master:port>
 ```
 
-If you use multiple Mesos masters:
-
-```
-docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 zk://<mesos_masters_url>
-```
+The command above creates a swarm manager listening at `<swarm_port>` and
+`<mesos_master_ip>` should point to where Mesos master lives in the cluster, 
+such as `localhost` or some IP address/hostname. `<public_machine_ip>` is the
+IP address for Mesos master to talk to swarm manager. If they are co-located on
+the same machine this value can be `0.0.0.0` or `localhost`.
 
-Once the manager is running, check your configuration by running `docker info` as follows:
+If you use multiple Mesos masters:
 
 ```
-docker -H tcp://<manager_ip:manager_port> info
+docker run -d -p <swarm_port>:2375 -p 3375:3375 \
+    swarm manage \
+        -c mesos-experimental \
+        --cluster-opt mesos.address=<public_machine_ip> \
+        --cluster-opt mesos.port=3375 \
+        zk://<mesos_masters_url>
 ```
 
-For example, if you run the manager locally on your machine:
+Once the manager is running, check your configuration by running `docker info`
+as follows:
 
 ```
+$ docker -H tcp://<manager_ip:manager_port> info
```

Comment:
just a nit, so only update if there are other comments; I think this should be `<manager_ip>:<manager_port>` to be consistent